### PR TITLE
Set background to white for pages and footer

### PR DESCRIPTION
**Pull request by SavvasStephanides**
UK feature branch: http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/opensdg_white_bg/

### CODE REVIEW

File: _sass/layouts/_footer.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 footer {
   clear: left;
-  background-color: #414042;
+  // background-color: #414042;
```
Comment (brockfanning) [inline _sass/layouts/_footer.scss]: I'm fine with removing the line altogether, to keep it clean.

File: _sass/variables/_colors.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 $main-background-color: #eaeaea;
```
Comment (brockfanning) [inline _sass/variables/_colors.scss]: Shouldn't this be removed too?

File: _sass/layouts/_footer.scss (hunk 1 above)
Comment (SavvasStephanides) [reply _sass/layouts/_footer.scss]: I agree

File: _sass/variables/_colors.scss (hunk 2 above)
Comment (SavvasStephanides) [reply _sass/variables/_colors.scss]: Removed

**Pull request merged**